utils.py

```diff
@@ -192,7 +192,8 @@ def get_num_mutations(seqs, parent_seq):
 
 
 def apply_mutations(parent,
-                    mutations):
+                    mutations,
+                    allow_same = False):
```

Comment:
how are conflicting mutations resolved when there are overlaps and allow_same=True? Can you update the docstring?